lib/src/model/movie_result.dart

```diff
@@ -42,21 +31,21 @@ class MoviesResult {
     }
 
     return MoviesResult(
-      voteCount: parsedJson['vote_count'],
       id: parsedJson['id'],
-      video: parsedJson['video'],
-      title: parsedJson['title'],
-      popularity: parsedJson['popularity'],
+      voteAverage: parsedJson['vote_average'] != null
```

Comment:
`parsedJson['vote_average'] ?? MovieStrings.defaultVoteAverage` do goo for inline null checks

Reply:
done